### PR TITLE
Add Export Win - Support Provided (step 5 of 6)

### PR DESCRIPTION
**Pull request by paulgain**
## Description of change

This is step 5 of 6 of the Export Win user journey that enables Lead Officers to add an Export Win to Data Hub.

## Test instructions

Go to `/exportwins/create?step=support_provided&company=<company-uuid>`

## Screen
Screenshot 2024-01-25 at 12 38 03
shots
## Validation
Screenshot 2024-01-25 at 12 39 52

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [ ] Has the branch been rebased to main?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

**Review by cgsunkel — APPROVED** [review]

**Review by peterhudec — CHANGES_REQUESTED** [review]
Looks good, just a couple of things which can be simplified.

File: src/client/modules/ExportWins/Form/SupportProvidedStep.jsx [hunk 1]
```diff
@@ -1,23 +1,92 @@
...
+          ? 'Choose a support type'
+          : value?.length > 5
+            ? 'Choose a maximum of 5 types of support'
+            : null
+      }
+    />
+    <ResourceOptionsField
+      name="associated_programme"
+      id="associated-programme"
+      label="What there as DBT campaign or event that contributed to this win?"
```
Comment (DeanElliott96) [inline src/client/modules/ExportWins/Form/SupportProvidedStep.jsx:53]: Think there is a typo here, might need to be rephrased?

**Review by DeanElliott96 — CHANGES_REQUESTED** [review]
Looks good, maybe just a typo in the label which needs rephrased I think.

**Review by peterhudec — APPROVED** [review]
Looks good now 💪

**Review by DeanElliott96 — APPROVED** [review]

**Pull request merged**